src/setuptools_scm/git.py

```diff
@@ -82,20 +83,23 @@ def fail_on_shallow(wd):
         )
 
 
-def parse(root, describe_command=DEFAULT_DESCRIBE, pre_parse=warn_on_shallow):
+def parse(root, config=None, describe_command=DEFAULT_DESCRIBE, pre_parse=warn_on_shallow):
```

Comment:
due to legacy consideration for now we have to add config as a argument at the end
while i dont expect to break any code, its still a useful precaution for users calling without keyword args

Reply:
Will do. Didn't see this as an external API.